eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedView.scala

```diff
@@ -415,13 +422,27 @@ trait EventsourcedView extends Actor with Stash {
     }
     case ReplaySuccess(events, progress, iid) => if (iid == instanceId) {
       events.foreach(receiveEvent)
+      // reset retry attempts
+      context.become(initiating(settings.replayRetryMax))
       replay(progress + 1L)
     }
-    case ReplayFailure(cause, iid) => if (iid == instanceId) {
-      logger.error(cause, s"replay failed, stopping self")
-      Try(onRecovery(Failure(cause)))
-      context.stop(self)
+    case ReplayFailure(cause, progress, iid) => if (iid == instanceId) {
+      if (replayAttempts < 1) {
+        // all replay attempts exceeded -> stop the actor
```

Comment:
*maximum number of replay attempts reached*? Is that comment needed at all? It just repeats what the `logger.error` in the next line says